tests/integration/tier0/non-destructive/firewalld-inhibitor/test_firewalld_inhibitor.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+
+@pytest.mark.test_firewalld_inhibitor
+def test_firewalld_inhibitor(shell, convert2rhel):
+    """
+    Verify that on the OL8.8 the conversion is inhibited due to
+    running firewalld on the system.
+    The reference ticket: https://issues.redhat.com/browse/RHELC-1180
+    """
+    assert shell("rpm -q firewalld").returncode == 0
+    assert shell("systemctl start firewalld").returncode == 0
+
+    with convert2rhel("-y --no-rpm-va --debug", unregister=True) as c2r:
+        c2r.expect(
+            "CHECK_FIREWALLD_AVAILABILITY::FIREWALLD_DAEMON_RUNNING - Firewalld is running",
+            timeout=600,
+        )
+
+    assert c2r.exitstatus == 1
+
+    # Clean up
+    assert shell("systemctl stop firewalld").returncode == 0
```

Comment:
I'd say, that cleaning up the system would be having the firewalld service running and enabled, since that is a default state of the image we test.
Considering these are non-destructive tests, we will never get past the point of no return, therefore the service won't cause the known issue.
On the other hand, if there is any yet undiscovered issue with the service before the PONR and/or any regression introduced, we should have the service running IMHO.
```suggestion

```

Reply:
I see your point, however, this "cleanup" should not happen in this test. What I mean is that the test should return to the state it was before the actual test, which was that the `firewalld` was stopped.

Reply:
Since we will remove the whole case in a foreseeable future, I'd say leave it this way.